src/Makefile

```diff
@@ -17,24 +28,44 @@ export CPU=32
 # MODEL_FOENIX_A2560U         9
 # MODEL_FOENIX_A2560K         13
 export MODEL=9
+ifeq ($(MODEL),9)
+	export MACHINE_TYPE=MACHINE_TYPE_A2560U
```

Comment:
I think these make for easier use in the code since it's added as a define by a -D, you can do just this in the code
```
#ifdef MACHINE_TYPE_A2560U
#endif
```

It looks a bit simpler than `#if MODEL == MODEL_FOENIX_A2560K` but it can be a matter of preference. What do you think ?